chef_master/source/automate_compliance_reporting.rst

```diff
@@ -29,18 +29,18 @@ Search Bar and Filters
 
 .. image:: ../../images/automate_compliance_searchbar.png
 
-The **Compliance** search allows you to view and filter compliance scan results based on a defined set of filters. 
+The **Compliance** search allows you to view and filter compliance scan results based on a defined set of filters.
 
 Filters
 ---------------------------------------------------------------
 * **Profile:** A profile is a set of controls expressed in the Chef InSpec language that define the preferred configuration of your system.  Chef Automate Compliance currently provides 88 built-in profiles under the `Profiles Store <profile_store.html>`__ You can also upload your own custom profiles expressed using the Chef InSpec language.
-* **Node:** A node is any machine that is under management by Chef.
+* **Node:** A node is any system that is under management by Chef.
```

Comment:
I'm struggling with this one, as a part-to-the-whole relationship. A node is part of a system, rather than the entire system.

Reply:
Machine is such awful slang for system though.